### DISCUSSION TITLE
running game NEED to reflect current scene's real-time state EVEN not saved

### DISCUSSION THREAD

**Discussion by jzq740176597** [godotengine/godot-proposals#6601] [Editor]
### Describe the project you are working on

3d

### Describe the problem or limitation you are having in your project

I just refine the scene (3d), I iterate to **change /amend some-setting** Then see what's effect or look like in game by running F5;
But I **have to** save the changed (setting can make it auto-done) If not what I want I Need to ctrl + Z sevaral times. It's so tedious.

Just see what's unity deal with this; game view just to reflect latest changed / state , save change or not it's up to user.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

If game view can reflect the latest state of scene (include the unsave-change) . I can easily amend or iterate changed sth. I may changed many If I want save then just Ctrl + S otherwise just reload it (discard the changes) then everything back to original state.
It's clear & fine

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

*

### If this enhancement will not be used often, can it be worked around with a few lines of script?

no

### Is there a reason why this should be core and not an add-on in the asset library?

core

**Comment by Zireael07** [comment]
Requested a ton of times, the last time like 2 days ago?

EDIT: And by yourself, and you were told it's a dupe of https://github.com/godotengine/godot-proposals/issues/1864

**Comment by Calinou** [comment]
It's already feasible to make changes to the running project that are **not** persisted by using the Remote scene tree dock while the project is running.
